### PR TITLE
Adds Hydrology coverage to Rasdaman

### PR DESCRIPTION
**Pull request by BobTorgerson**
This PR adds the Hydrology dataset as a Rasdaman coverage. To test, run this ingest script on Apollo with a new coverage title.

Additionally, be sure to read the README in this directory and follow the BASH commands in there to ensure you can get a clean ingest. 

Here is where the data exists: `poseidon.snap.uaf.edu:/workspace/Shared/Tech_Projects/rasdaman_production_datasets/hydrology`

### CODE REVIEW

File: ardac/hydrology/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Hydrology Rasdaman coverage ingests
+
+To run this ingest without the need to restart it over again, you will want to run this ingest using the BASH script included in this directory. It will run the ingest and restart the ingest if it does not get a good exit code from the command to allow for a single command to ensure the ingest process finishes.
```
Comment (charparr) [inline ardac/hydrology/README.md:3]: Thank you Bob!! I wonder if the Rasdaman developers saw this if they'd dig into the bug, or ask you to submit a PR for this script 😁

File: ardac/hydrology/luts.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    "dec": 2,
+    "feb": 3,
+    "jan": 4,
+    "jul": 5,
+    "jun": 6,
+    "mar": 7,
+    "may": 8,
+    "nov": 9,
+    "oct": 10,
+    "sep": 11,
```
Comment (charparr) [inline ardac/hydrology/luts.py:46]: I feel like this embodies the hard-won Rasda-knowledge: don't chrono the months, they have to be alphabetized!

File: ardac/hydrology/run_ingest.sh [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+source /etc/profile.d/rasdaman.sh
+
+while true; do
+    /opt/rasdaman/bin/wcst_import.sh -c 0 ingest.json 2>&1   
+
+    # Get the exit status of the last command
+    exit_status=$?
+
+    # Check if the exit status is 0
```
Comment (charparr) [inline ardac/hydrology/run_ingest.sh:11]: Love little shell scripts like this! I'm just noting that if it actually started building a coverage this script may need to be modified to flush that coverage prior to attempting another re-ingest. This script could save us from some ingest baby-sitting!

**Review by charparr — APPROVED** [review]
I successfully ran this ingest on Apollo and created a coverage called `cp_test_hydrology`. I copied the GeoTIFF data directory from `usr/local/data/torgerso/correct_decades_zipped/` to a directory that I created in `/tmp` on Apollo and brought the ingest and luts files in there. After setting the LUTS path (`export LUTS_PATH=/tmp/cp_hydrology/luts.py`) I began the ingest (`opt/rasdaman/bin/wcst_import.sh ingest.json`) which completed with zero issues. I didn't time this, but it maybe took 3ish hours.

**Review by kyleredilla — APPROVED** [review]
I ran this ingest with no issues and successfully created a coverage! I did time it after seeing Charlie's comment, took 2.8 hrs. That script for babysitting the ingest worked great.

File: ardac/hydrology/luts.py (hunk 2 above)
Comment (brucecrevensten) [reply ardac/hydrology/luts.py:28]: or just use numbers for months instead of names

**Pull request merged**